### PR TITLE
[Portuguese] Update tags in all terms

### PR DESCRIPTION
**Pull request by edsoncelio**
Signed-off-by: edsoncelio <edsoncelio2020@gmail.com>

### Describe your changes
Update tags in all terms localized in Portuguese (Brazil).

### Related issue number or link (ex: `resolves #issue-number`)
https://github.com/cncf/glossary/pull/1583

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by brunoguidone — APPROVED** [review]
LGTM

**Review by MrErlison — APPROVED** [review]

**Comment by edsoncelio** [comment]
PTAL @seokho-son @jihoon-seo

File: content/pt-br/data-center.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Data Center
 status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/pt-br/data-center.md:3]:
```suggestion
status: Feedback Appreciated
draft: true
```

File: content/pt-br/firewall.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Firewall
-status: Completed
+status: Deprecated
```
Comment (jihoon-seo) [inline content/pt-br/firewall.md:3]:
```suggestion
status: Deprecated
draft: true
```

File: content/pt-br/platform-as-a-service.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Plataforma como serviço (PaaS)
-status: Completed
+status: Deprecated
```
Comment (jihoon-seo) [inline content/pt-br/platform-as-a-service.md:3]:
```suggestion
status: Deprecated
draft: true
```

File: content/pt-br/software-as-a-service.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 Title: Software como Serviço
-Status: Completed
+Status: Deprecated
```
Comment (jihoon-seo) [inline content/pt-br/software-as-a-service.md:3]:
```suggestion
Status: Deprecated
draft: true
```

File: content/pt-br/stateless-apps.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 title: Aplicações Stateless
 status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/pt-br/stateless-apps.md:3]:
```suggestion
status: Feedback Appreciated
draft: true
```

File: content/pt-br/version-control.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Controle de Versão
-status: Completed
+status: Deprecated
```
Comment (jihoon-seo) [inline content/pt-br/version-control.md:3]:
```suggestion
status: Deprecated
draft: true
```

File: content/pt-br/debugging.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 title: Debugging
 status: Completed
 category: conceito
```
Comment (jihoon-seo) [inline content/pt-br/debugging.md:4]:
```suggestion
status: Deprecated
draft: true
category: conceito
```

**Review by jihoon-seo — COMMENTED** [review]
Recently,
- the front-matter `draft: true` has been added to some English terms in #1570.
- some terms' status has been modified in #1449 and other PRs.

PTAL again! 😊

**Comment by edsoncelio** [comment]
Closing due to the number of changes - I'll open a new one soon :D

**Pull request closed without merging**